src/backend/interfaces/REST/go.server/operations/devices/devices.go

```diff
@@ -8,24 +8,25 @@ package devices
 
 import (
 	"bytes"
-	"errors"
-	"fmt"
 	. "github.com/CaliOpen/Caliopen/src/backend/defs/go-objects"
 	"github.com/CaliOpen/Caliopen/src/backend/interfaces/REST/go.server/middlewares"
 	"github.com/CaliOpen/Caliopen/src/backend/interfaces/REST/go.server/operations"
 	"github.com/CaliOpen/Caliopen/src/backend/main/go.main"
 	"github.com/gin-gonic/gin"
-	"github.com/gin-gonic/gin/binding"
 	swgErr "github.com/go-openapi/errors"
-	"github.com/satori/go.uuid"
 	"io/ioutil"
 	"net/http"
 	"strconv"
-	"strings"
 )
 
 // NewDevice handles POST /devices
 func NewDevice(ctx *gin.Context) {
+	e := swgErr.New(http.StatusForbidden, "do not create new device this way")
```

Comment:
remove the route from swagger, it will raise a 404, a forbidden is not a valid response